app/vulncode_app/models.py

```diff
@@ -73,6 +76,7 @@ class Package(models.Model):
     A software package with minimal identifying information.
     Other identifiers are stored as PackageReference.
     """
+    impacted_package = models.ManyToManyField('ImpactedPackage')
     platform = models.CharField(max_length=50, help_text='Package platform eg:maven', blank=True)
     name = models.CharField(max_length=50, help_text='Package name', blank=True)
     version = models.CharField(max_length=50, help_text='Package version', blank=True)
```

Comment:
in the line below 
```
def __str__(self):
    return self.name
```
we likely need more than just name ... , but this is minor